azure_jumpstart_ag/artifacts/PowerShell/AgConfig-manufacturing.psd1

```diff
@@ -220,9 +216,9 @@
 
     AppConfig = @{
         inferencing_deployment = @{
-            GitOpsConfigName = "deploy-inferencing"
-            KustomizationName = "inferencing"
-            KustomizationPath="./contoso_manufacturing/operations/contoso_manufacturing/releases/inferencing"
```

Comment:
I think we need to change `contoso_manufacturing` to `contoso_motors` on the jumpstart-agora-apps repo and everywhere here as well

Reply:
this will require another PR against the other repo. Before we do that I dont want to interrupt other work on that repo with a path change. Can we pause on this change for now?

Reply:
agreed